components/select/README.md

```diff
@@ -22,6 +22,26 @@ var vm = new Vue({
 > native rendering based on the media query
 > `(max-width: 600px) and (pointer: coarse)`.
 
+### Events
+
+#### mdc-select
+
+| event | args | Description |
+|-------|------|-------------|
+|`@blur`||emitted on select blur |
+
+#### mdc-multi-select
```

Comment:
plz do remove doc for mdc-multi-select and mdc-native-select. Those are internal components. 
Only mdc-select and mdc-option should be documented here.